src/Providers/Trustpilot/ServiceActions.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace PodPoint\Reviews\Providers\Trustpilot;
+
+use PodPoint\Reviews\ApiClientInterface;
+use PodPoint\Reviews\Exceptions\ValidationException;
+use PodPoint\Reviews\Providers\Trustpilot\Request\Service\FindReviewRequest;
+use PodPoint\Reviews\Providers\Trustpilot\Request\Service\InviteRequest;
+use PodPoint\Reviews\Providers\Trustpilot\Request\Service\GetReviewsRequest;
+use PodPoint\Reviews\ActionsInterface;
+
+/**
+ * Class ServiceActions
+ * @package PodPoint\Reviews\Providers\Trustpilot
+ */
+class ServiceActions implements ActionsInterface
+{
+    /**
+     * @var ApiClientInterface
+     */
+    protected $apiClient;
+
+    /**
+     * @var string
+     */
+    protected $businessUnitId;
+
+    /**
+     * ServiceActions constructor.
+     * @param ApiClientInterface $apiClient
+     */
+    public function __construct(ApiClientInterface $apiClient)
+    {
+        $this->apiClient = $apiClient;
+    }
+
+    /**
+     * Invite consumers.
+     *
+     * @param array $options
+     * @return mixed
+     * @throws ValidationException
+     */
+    public function invite(array $options)
+    {
+        $options['businessUnitId'] = $this->businessUnitId;
+        $request = new InviteRequest($this->apiClient, $options);
+
+        return $request->send();
+    }
+
+    /**
+     * Get reviews.
+     *
+     * @param array $options
+     *
+     * @return mixed
+     * @throws ValidationException
+     */
+    public function getReviews(array $options)
```

Comment:
I have not tested this yet but I assume same point here about making default value of array [] to prevent error if calling function without passing param